config/pghero.yml

```diff
@@ -0,0 +1,52 @@
+databases:
+  primary:
+    # Database URL
+    url: <%= ENV.fetch('PGHERO_DATABASE_URL', Rails.env.test? ? nil : fail('Missing PGHERO_DATABASE_URL')) %>
```

Comment:
Requiring this in development or production also causes problems. Fixed (hopefully) here: https://github.com/davidrunger/david_runger/pull/6428 .